source/MaterialXRenderGlsl/GlslValidator.cpp

```diff
@@ -398,10 +398,10 @@ void GlslValidator::validateRender()
     glViewport(0, 0, _frameBufferWidth, _frameBufferHeight);
 
     // Update viewing information
-    const Vector3 eye(0.0f, 0.0f, 40.0f);
+    const Vector3 eye(0.0f, 0.0f, 4.0f);
     const Vector3 center;
     const Vector3 up(0.0f, 1.0f, 0.0f);
-    float objectScale(10.0f);
+    float objectScale(1.0f);
```

Comment:
This attempt to scale the scene to match the OSL scene doesn't seem to work correctly. We can investigate what the problem is but as a quite fix we have the scene data without any scaling for now. The scaling of 10 units also made the light sources placed inside the sphere.